test/unit/enrichment/enrich-invoice-line.test.js

```diff
@@ -56,13 +54,25 @@ describe('enrich header', () => {
 
   test('should retain convergence if already set', async () => {
     invoiceLine.convergence = true
-    await enrichInvoiceLine(invoiceLine, fundCode)
+    await enrichInvoiceLine(invoiceLine, scheme)
     expect(invoiceLine.convergence).toBe(true)
   })
 
   test('should set convergence to false if not already set', async () => {
     delete invoiceLine.convergence
-    await enrichInvoiceLine(invoiceLine, fundCode)
+    await enrichInvoiceLine(invoiceLine, scheme)
     expect(invoiceLine.convergence).toBe(false)
   })
+
+  test('should retain delivery body if already set', async () => {
+    const originalDeliveryBody = invoiceLine.deliveryBody
+    await enrichInvoiceLine(invoiceLine, scheme)
+    expect(invoiceLine.deliveryBody).toBe(originalDeliveryBody)
+  })
+
+  test('should set delivery body if not already set and delivery body defined', async () => {
```

Comment:
Maybe change the name of this test to something like 'should set delivery body to defined delivery body is not already set'? Does that make it more clear?

Reply:
I was replicating the language used for the existing fund code tests for consistency.

Reply:
Yeah that makes sense - that's fine then